runtime/src/bank.rs

```diff
@@ -622,6 +622,12 @@ pub struct Bank {
     pub feature_set: Arc<FeatureSet>,
 }
 
+impl Default for Bank {
+    fn default() -> Self {
+        Self::new(&GenesisConfig::default())
+    }
+}
+
```

Comment:
I _think_ this was a safe move.  `Bank::default()` seemed to only be used in tests/benches and the derive was making the changes of `feature_builtins` to `Arc<[(Builtin, Pubkey)]` a PITA since `Entrypoint` has no sane default value.